conversion.md

```diff
@@ -0,0 +1,89 @@
+# Conversion to OCI Runtime Configuration
+
+When extracting an OCI Image into an [OCI Runtime bundle][oci-runtime-bundle], two components of the orthogonal components of the extraction are relevant:
```

Comment:
“components of the orthogonal components of the extraction” → “orthogonal components”?